KsApi/models/User.swift

```diff
@@ -16,6 +16,7 @@ public struct User {
   public var showPublicProfile: Bool?
   public var social: Bool?
   public var stats: Stats
+  public var unseenActivityCount: Int?
```

Comment:
Optional so that we don't break the deserialisation of users stored in `UserDefaults`.